README.md

```diff
@@ -1,4 +1,4 @@
-# Dependency Injection Graph
+# dig - A dependency injection framework for Go
```

Comment:
Yeah, that works too. Eventually we can get @Lupie to help with this i'm sure she would immediately know what to do :)